python/lsst/pipe/drivers/utils.py

```diff
@@ -1,7 +1,15 @@
-from lsst.pipe.base import Struct
```

Comment:
This file has some linter warnings and even one error (undefined name "argparse"). Please fix before merging to master.

Preferably run all files in this package through a linter (flake8 with suitable overrides, or pylint) before merging.